src/pages/getting_started/first_app.md

```diff
@@ -475,6 +475,19 @@ Please visit [Debugging App Builder Apps Codelab](../resources/debugging/index.m
 
 If the local development is run without the `--local` option, the action you are debugging is running directly on [Adobe I/O Runtime](/apis/experienceplatform/runtime). When the `--local` option is used, the actions are debugged in the standalone OpenWhisk instance running locally. In both cases, Docker is required on your local machine so that the remote or locally mounted actions get proxied in the local Docker container. Learn more about what you can do in [debug mode](https://www.npmjs.com/package/@openwhisk/wskdebug).
 
+#### Targeting local actions with `aio runtime` 
+To target local actions with `aio runtime`, you will have to populate the project's `.env` with the local Runtime credentials. 
+
+These credentials can be found in `dist/<extension|application>/.env.local`. 
```

Comment:
This might be an issue since this file is not written unless we run `aio app run --local` (anything in dist are artifacts).

Maybe we just list the local credentials in the docs? https://github.com/adobe/aio-cli-plugin-app/blob/master/src/lib/owlocal.js

Reply:
Great idea. New commit incoming